qualtran/bloqs/chemistry/hubbard_model/qubitization/select_hubbard_test.py

```diff
@@ -35,7 +35,7 @@ def test_select_t_complexity(dim):
     assert cost == GateCounts(
         cswap=2 * logN, and_bloq=5 * (N // 2) - 2, measurement=5 * (N // 2) - 2, clifford=ANY
     )
-    assert cost.total_t_count() == 10 * N + 14 * logN - 8
+    assert cost.total_t_count() == 10 * N + 8 * logN - 8
```

Comment:
What does the linear T paper quote again / why does this change?

Reply:
per above, we assert cswap=2*logN. This changes the T count from 7*2*logN to 4*2*logN, so I think it's legit